### PR TITLE
Add pull request template

### PR DESCRIPTION
**Pull request by saraedum**

### CODE REVIEW

**Comment by saraedum** [comment]
I don't deem this news-worthy and there's nothing to test :)

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!--
+Thank you for pull request.
```
Comment (slel) [inline .github/pull_request_template.md:2]: Thank you for submitting a pull request.

**Review by slel — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md (hunk 1 above)
Comment (saraedum) [reply .github/pull_request_template.md:2]: Lol, I copied this from conda-forge's pull request template, so I didn't even proof read that part:

image

@isuruf, were you aware of this error? Where can this be fixed?

**Comment by slel** [comment]
Really, no need to change it. It goes straight to the point.
How to choose between "for working on a pull request",
"for this pull request", "for your pull request", etc.  : )

File: .github/pull_request_template.md (hunk 1 above)
Comment (isuruf) [reply .github/pull_request_template.md:2]: Source is at https://github.com/conda-forge/.github/blob/master/.github/PULL_REQUEST_TEMPLATE.md

Comment (saraedum) [reply .github/pull_request_template.md:2]: Thanks. Fixed in https://github.com/conda-forge/.github/pull/8/files.

**Pull request merged**